### PR TITLE
Update secure-score-improvement-action-details.png

### PR DESCRIPTION
**Pull request by msbemba**
Changed screenshot for improvement action as it has changed.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7778

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ When you select a specific improvement action, a full page flyout appears.
 
 To complete the action, you have a few options:
 
-- Select **Manage** to go the configuration screen and make the change. You'll then gain the points that the action is worth, visible in the fly out. Points generally take about 24 hours to update.
+- Select **Manage in Microsoft 365 Defender** to go the configuration screen and make the change. You'll then gain the points that the action is worth, visible in the fly out. Points generally take about 24 hours to update.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/microsoft-secure-score-improvement-actions.md]:
```suggestion
- Select **Manage in Microsoft 365 Defender** to go to the configuration screen and make the change. You'll then gain the points that the action is worth, visible in the flyout. Points generally take about 24 hours to update.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Kindly merge this PR. Thanks

**Pull request merged**